lib/groups-spike.js

```diff
@@ -0,0 +1,194 @@
+/* globals LastReadDeprecated, FollowDeprecated */
```

Comment:
Is this file going to be left in the code? If so, is there a better name for it?

Reply:
OK, I can move it somewhere into api/models/group as a util or migration file.